controller/codebase.go

```diff
@@ -187,13 +187,7 @@ func (c *CodebaseController) Open(ctx *app.OpenCodebaseContext) error {
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
 	}
 	cheClient := che.NewStarterClient(c.config.GetCheStarterURL(), c.config.GetOpenshiftTenantMasterURL(), getNamespace(ctx))
-	workspace := che.WorkspaceRequest{
-		Name:       ctx.WorkspaceID,
-		Repository: cb.URL,
-		Branch:     "master",
-		StackID:    cb.StackID,
-	}
-	workspaceResp, err := cheClient.CreateWorkspace(ctx, workspace)
+	workspaceResp, err := cheClient.StartExistingWorkspace(ctx, ctx.WorkspaceID)
```

Comment:
@aslakknutsen correct, we only expected to that platform will start workspace without changing other properties like description. As part of https://github.com/redhat-developer/che-starter/issues/101 it is planned to process path & line parameters from the body